### DISCUSSION TITLE
From which localisation platform are locale files retrieved?

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [guardianproject/orbot-android#1136] [Q&A]
Good morning. As it appears, two localisation platforms are currently selected to host locale files; _Transifex_, _Weblate_. Given that these files can only be maintained by a single localisation platform at a time, it remains to be determined which of these platforms is the one from which these files are repatriated to this repository?

**Comment by n8fr8** [comment]
Weblate is the official: https://hosted.weblate.org/projects/guardianproject/orbot/

That said, we can merge contributions from both, and in some cases we have users that prefer Transifex still.
